src/test/java/org/opensearch/neuralsearch/query/HybridQueryIT.java

```diff
@@ -94,17 +108,24 @@ public void testBasicQuery_whenOneSubQuery_thenSuccessful() {
         assertEquals(3, getHitCount(searchResponseAsMap1));
 
         List<Map<String, Object>> hits1NestedList = getNestedHits(searchResponseAsMap1);
-        List<String> ids1 = new ArrayList<>();
-        List<Double> scores1 = new ArrayList<>();
+        List<String> ids = new ArrayList<>();
+        List<Double> scores = new ArrayList<>();
         for (Map<String, Object> oneHit : hits1NestedList) {
-            ids1.add((String) oneHit.get("_id"));
-            scores1.add((Double) oneHit.get("_score"));
+            ids.add((String) oneHit.get("_id"));
+            scores.add((Double) oneHit.get("_score"));
         }
 
         // verify that scores are in desc order
-        assertTrue(IntStream.range(0, scores1.size() - 1).noneMatch(idx -> scores1.get(idx) < scores1.get(idx + 1)));
+        assertTrue(IntStream.range(0, scores.size() - 1).noneMatch(idx -> scores.get(idx) < scores.get(idx + 1)));
         // verify that all ids are unique
-        assertEquals(Set.copyOf(ids1).size(), ids1.size());
+        assertEquals(Set.copyOf(ids).size(), ids.size());
+
+        Map<String, Object> total = getTotalHits(searchResponseAsMap1);
+        assertNotNull(total.get("value"));
+        assertEquals(3, total.get("value"));
```

Comment:
Would be nice to have `3` as a variable.

Reply:
ack